tests/connector-service-tests/README.md

```diff
@@ -2,53 +2,36 @@
 
 ## Overview
 
-This project contains the acceptance tests for Connector Service.
-The tests are written in Go. Run them as standard Go tests.
+This project contains the acceptance tests for Kyma Connector Service.
+
+## Prerequisites
+
+The Connector Service Acceptance Tests requires Go 1.8 or higher.
+
+## Build
+
+To install the Metadata Service components, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+2. `cd /tests/connector-service-tests`
+3. `CGO_ENABLED=0 go build ./test/apitests/connector_test.go`
 
 ## Usage
 
-This section provides information on building and versioning of the Docker image, as well as configuring the Kyma.
-
-### Running locally
-
-1. `helm ls` and pick a release you want to test.
-```sh
-$ helm ls
-NAME               	REVISION	UPDATED                 	STATUS  	CHART                        	NAMESPACE
-cluster-essentials 	1       	Tue May 22 10:47:14 2018	DEPLOYED	kyma-cluster-essentials-0.0.1	kyma-system
-core               	2       	Tue May 22 11:00:03 2018	DEPLOYED	core-0.0.1                   	kyma-system
-ec-default         	1       	Tue May 22 11:00:24 2018	DEPLOYED	gateway-0.0.1                	kyma-integration
-hmc-default        	1       	Tue May 22 11:00:07 2018	DEPLOYED	gateway-0.0.1                	kyma-integration
-istio              	1       	Tue May 22 10:47:21 2018	DEPLOYED	istio-0.5.1                  	istio-system
-prometheus-operator	1       	Tue May 22 10:48:10 2018	DEPLOYED	prometheus-operator-0.18.1   	kyma-system
-```
-
-2. Let's say we pick `ec-default` release. In order to test it, run `helm test ec-default`.
-```sh
-$ helm test ec-default
-RUNNING: test-ec-default-acceptance
-PASSED: test-ec-default-acceptance
-```
-
-3. To see the logs, find the pod, that ran tests using `kubectl get po --show-all`. It'll be in `Completed` state.
-```sh
-$ kubectl get po --show-all
-NAME                                  READY     STATUS      RESTARTS   AGE
-ec-default-gateway-5dffff49f-drr7n    2/2       Running     0          1h
-hmc-default-gateway-669965894-5wwv8   2/2       Running     0          3h
-test-ec-default-acceptance            0/1       Completed   0          7m # <<<<<< this one
-```
-
-and then just
-
-```sh
-kci logs test-ec-default-acceptance
-=== RUN   TestGatewayHealth
-=== RUN   TestGatewayHealth/SF_Gateway
---- PASS: TestGatewayHealth (0.00s)
-    --- PASS: TestGatewayHealth/SF_Gateway (0.00s)
-=== RUN   TestApiMetadata
---- PASS: TestApiMetadata (0.00s)
-PASS
-ok  	github.com/kyma-project/gateway-tests/test	0.003s
-```
+### Environment parameters
+
```

Comment:
Are all of them obligatory? Perhaps it would be better to add a table similar to the one [here](https://github.com/kyma-project/kyma/tree/master/components/api-controller#use-environment-variables)?

Either way, pls add some sort of description above the list, e.g. `Use these environment variables to configure the application:`